src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -199,6 +211,23 @@ def __init__(self, target: Text,
         if compression:
             raise NotImplementedError("TODO: compression not implemented yet")
 
+        if interceptors is None:
+            self._unary_unary_interceptors = None
+        else:
+            self._unary_unary_interceptors = list(
+                filter(
+                    lambda interceptor: isinstance(interceptor, UnaryUnaryClientInterceptor),
+                    interceptors)) or None
+
+            invalid_interceptors = set(interceptors) - set(
+                self._unary_unary_interceptors or [])
+
+            if invalid_interceptors:
```

Comment:
We will be extending later with other interceptors the previous expression, like for example

```invalid_interceptors = set(interceptors) - set(
                self._unary_unary_interceptors) - set(self._unary_stream_interceptors)
```

With this, we are only raising a `ValueError` if and only if one of the interceptors does not meet with one of the supported classes.

Because we do not support for now any other interceptors, for now the check is only considering the unary unary ones.